app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/ActivityTrackingController.java

```diff
@@ -51,14 +52,15 @@
 import io.syndesis.common.util.KeyGenerator;
 import io.syndesis.server.jsondb.GetOptions;
 import io.syndesis.server.jsondb.JsonDB;
+import io.syndesis.server.jsondb.impl.JsonRecordSupport;
 import io.syndesis.server.openshift.OpenShiftService;
 
 /**
  * This class tracks pod controller and ingests them into our DB.
  */
 @Service
 @ConditionalOnProperty(value = "controllers.dblogging.enabled", havingValue = "true", matchIfMissing = true)
-@SuppressWarnings({"PMD.DoNotUseThreads", "PMD.ModifiedCyclomaticComplexity", "PMD.StdCyclomaticComplexity", "PMD.CyclomaticComplexity"})
+@SuppressWarnings({"PMD.DoNotUseThreads", "PMD.ModifiedCyclomaticComplexity", "PMD.StdCyclomaticComplexity", "PMD.CyclomaticComplexity", "PMD.GodClass"})
```

Comment:
Might be worth to split the class up at some point in time. At least this is commented ;-)